app/models/pubmed_source_record.rb

```diff
@@ -108,27 +108,28 @@ def extract_year_from_pubmed_record(publication)
   end
 
   # see https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/el-Month.html
-  def extract_month_from_pubmed_record(publication)
+  # see https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/el-Day.html
+  def extract_month_day_from_pubmed_record(publication)
     # look for a month in all of the xpath locations in order
```

Comment:
we now have a single method to return both a month and day together, instead of two separate methods